package.json

```diff
@@ -62,17 +67,20 @@
     "@typescript-eslint/eslint-plugin": "4.31.0",
     "@typescript-eslint/parser": "4.31.0",
     "babel-eslint": "10.1.0",
+    "cypress": "8.3.1",
     "eslint": "7.32.0",
     "eslint-config-next": "11.1.2",
     "eslint-config-prettier": "8.3.0",
     "eslint-import-resolver-typescript": "2.4.0",
+    "eslint-plugin-cypress": "2.11.3",
     "eslint-plugin-import": "2.24.2",
     "eslint-plugin-jsx-a11y": "6.4.1",
     "eslint-plugin-react": "7.25.1",
     "eslint-plugin-react-hooks": "4.2.0",
     "husky": "7.0.2",
     "lint-staged": "11.1.2",
     "prettier": "2.4.0",
+    "start-server-and-test": "1.14.0",
```

Comment:
What is the `start-server-and-test` package doing for us? Does it do something that we can not get using Cypress?

Reply:
You can't get that with Cypress, they actually recommend its usage: https://docs.cypress.io/faq/questions/using-cypress-faq#How-do-I-wait-for-my-application-to-load

Cypress will run the tests against an instance of the app, but Cypress itself doesn't take care of running the app for you. So you need to run `yarn start` in one terminal, and then `yarn cypress` in another one, so Cypress can run the tests against the app running locally.

What if you want to do everything at once (e.g. in CI)? You need to set this package to first start your app, wait for it, and then run Cypress against it.

Reply:
You're right, Cypress does recommend that package for use in the CI... 😄 Just checking! 🙈